quiche/src/lib.rs

```diff
@@ -1958,10 +1958,16 @@ impl<F: BufFactory> Connection<F> {
             peer,
             &recovery_config,
             config.path_challenge_recv_max_queue_len,
-            MIN_CLIENT_INITIAL_LEN,
             true,
         );
 
+        if config.pmtud {
+            path.pmtud = Some(pmtud::Pmtud::new(
+                MIN_CLIENT_INITIAL_LEN,
+                config.max_send_udp_payload_size,
```

Comment:
There's something odd here.  I thought that max_send_udp_payload_size was the size that the quiche implementation uses when sending out packets.

I would expect the pmtud probe size to be a larger configured value.  It can't be the same as config.max_send_udp_payload_size

Reply:
The source of my confusion is https://github.com/cloudflare/quiche/blob/d45788b/quiche/src/lib.rs#L5008

config.max_send_udp_payload_size acts as the send packet size when pmtud is disabled.  When pmtud is enabled, recovery.max_send_udp_payload_size is set to 1200 bytes by this code and then pmtud can try to raise it.